src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4794,8 +4805,21 @@ class RideWindow final : public Window
 
     void MusicResize()
     {
-        flags |= WF_RESIZABLE;
-        WindowSetResize(*this, 316, 81, 316, 81);
+        auto ride = GetRide(rideId);
+        if (ride != nullptr)
+        {
+            // Expand the window when music is playing
+            auto isMusicActivated = (ride->lifecycle_flags & RIDE_LIFECYCLE_MUSIC) != 0;
+            if (isMusicActivated)
+            {
+                // make sure to also change the min height value in the ToggleMusic() function when modifying this
+                WindowSetResize(*this, 316, 207, 500, 450);
+            }
+            else
+            {
+                WindowSetResize(*this, 316, 81, 316, 81);
```

Comment:
Why are so many dimensions changed? I thought only height would have to increase.

Reply:
My idea was to shrink the both width and height by changing the window max values when the toggle switch is off. I'm not too familiar with how the resizing works.

Maybe something like this for the else section?
```
height = 81;
width = 316;
WindowSetResize(*this, 316, 81, 500, 450);
```